sqlFile.js

```diff
@@ -0,0 +1,43 @@
+var sqlite3 = require("sqlite3").verbose();
+var db = new sqlite3.Database("url-shortner.db");
+
+const DBSOURCE = "db.sqlite3";
+
+function createUrlTable() {
+  db.run("CREATE TABLE IF NOT EXISTS urlTable (fullUrl TEXT, shortCode TEXT)");
+}
+
+function gettingTableValues(fullUrl, shortCode) {
```

Comment:
A name like `gettingTableValues` doesn't really make sense for this function. Use clear names that communicate the purpose of the function. In this case a name like `insertShortCode` or `addShortCode` might be better.